invenio_app_ils/cli.py

```diff
@@ -398,6 +410,16 @@ def generate(self):
             else:
                 issue += 1
 
+        demo_data_dir = os.path.join(
+            os.path.realpath("."), "invenio_app_ils", "demo_data"
+        )
+        with open(os.path.join(demo_data_dir, "documents.json")) as f:
+            demo_data = json.loads(f.read())
+
+        for doc in demo_data:
+            doc["pid"] = self.create_pid()
+
+        objs.extend(demo_data)
```

Comment:
do we still need the lorem ipsum documents? Same above

Reply:
I think we need them because the real documents are only 6.